utilities/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = { workspace = true }
 rust-version = { workspace = true }
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
consider adding `default-features = false` here for no_std support.

Reply:
I made `bytes_from_field_elements` infallible instead. Don't need `anyhow` dependency anymore so I removed it.